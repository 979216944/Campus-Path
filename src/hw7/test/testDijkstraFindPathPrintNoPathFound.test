# Test Dijkstra finding a path.
# Given two valid node arguments,
# that have no path in the specified graph.

# Load graph
LoadGraph flowers flowers.tsv

# Find a minimum-cost path with two valid node arguments,
# that have no path in the specified graph
FindPath flowers Alchemilla Verbena
FindPath flowers Verbena Alchemilla