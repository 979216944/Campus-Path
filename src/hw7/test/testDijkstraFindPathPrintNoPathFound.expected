# Test Dijkstra finding a path.
# Given two valid node arguments,
# that have no path in the specified graph.

# Load graph
loaded graph flowers

# Find a minimum-cost path with two valid node arguments,
# that have no path in the specified graph
path from Alchemilla to Verbena:
no path found
path from Verbena to Alchemilla:
no path found